docker-compose.dev.yml

```diff
@@ -22,5 +22,5 @@ services:
       - POSTGRES_PASSWORD=postgres
     ports:
       - '5432:5432'
-    volumes:
-      - ./postgresdata:/var/lib/postgresql/data
\ No newline at end of file
+#    volumes:
```

Comment:
Rather delete the lines than commenting them

Reply:
yes, reverted commenting.

Reply:
should be already fixed here https://github.com/bitcoin-sv/arc/pull/120/files#diff-9542f82d64bbeebd91f6236324bfe199e9657e2cb1fd9779d5d6dcdcf9cd4de1